engine/src/main/java/org/camunda/bpm/engine/impl/persistence/entity/ProcessDefinitionManager.java

```diff
@@ -164,9 +165,8 @@ protected ProcessDefinitionEntity findProcessDefinitionByKeyVersionOrVersionTag(
   }
 
   public List<ProcessDefinition> findProcessDefinitionsByKey(String processDefinitionKey) {
-    ProcessDefinitionQueryImpl processDefinitionQuery = new ProcessDefinitionQueryImpl()
-      .processDefinitionKey(processDefinitionKey);
-    return  findProcessDefinitionsByQueryCriteria(processDefinitionQuery, null);
+    ProcessDefinitionQueryImpl processDefinitionQuery = new ProcessDefinitionQueryImpl().processDefinitionKeysIn(processDefinitionKey);
```

Comment:
**Note for future reference**: the query will be different after this change is applied. However, we don't use this method so far. Therefore there's no impact considering performance etc. (also this is part of the private API). I guess the method was added back then for completeness and option to be used if necessary.